tests/test_downloads.py

```diff
@@ -143,7 +143,9 @@ def test_download_cached(self, export, get_cached):
         assert not export.delay.called
 
     def test_download_forbidden(self):
-        with pytest.raises(ApiError):
+        # should the tests throw exceptions?
```

Comment:
The test is asserting that a particular exception is raised. Nothing wrong with that.